test/steps/suite_steps.go

```diff
@@ -45,10 +46,13 @@ func SuiteSteps(suite *godog.Suite, fs *FeatureState) {
 	})
 
 	suite.AfterScenario(func(args interface{}, e error) {
-		// remove the GitEnvironment of the scenario
-		err := fs.activeScenarioState.gitEnvironment.Remove()
-		if err != nil {
-			log.Fatalf("error removing the Git environment after scenario %q: %v", scenarioName(args), err)
+		if e != nil {
```

Comment:
Thoughts on flipping the if / else so this can be
```go
if e == nil {
  //...
} else {
  //...
}
```
I think that makes it a bit simpler.

Reply:
done